gcp/build.gradle

```diff
@@ -70,6 +72,51 @@ tasks.register('deployToGCE') {
     }
 }
 
+tasks.register('jupyterGCE') {
+    doLast {
+        def propsFileName = "../gradle.deploy.properties"
+        if (project.hasProperty('propsFile')) {
+            propsFileName = project.findProperty('propsFile')
+        }
+
+        def propsFile = new Properties()
+        propsFile.load(project.file(propsFileName).newDataInputStream())
+
+        ext.getParameterValue = { paramName ->
+            if (project.hasProperty(paramName)) {
+                return project.findProperty(paramName)
+            } else {
+                return propsFile.getProperty(paramName)
+            }
+        }
+
+        if (!ext.getParameterValue('runName')) {
+            throw new GradleException('Please name the run by specifying `runName` argument. e.g; ./gradlew deploy -PrunName=sfbay-performance-run')
+        }
+        def tempInstanceType = "${ext.getParameterValue('instanceType') ?: 'n2-highmem-4'}"
+        def finalInstanceType = tempInstanceType.isEmpty() ? null : tempInstanceType
+        GString pload = """{
+        "run_name": "${(ext.getParameterValue('runName') ?: 'jupyter')  + '_' + getCurrentGitUserEmail()}",
+        "instance_type": "${finalInstanceType}",
+        "forced_max_ram": "${ext.getParameterValue('forcedMaxRAM')}",
+        "beam_branch": "${ext.getParameterValue('beamBranch') ?: getCurrentGitBranch()}",
+        "beam_commit": "${ext.getParameterValue('beamCommit') ?: 'HEAD'}",
+        "data_branch": "${ext.getParameterValue('dataBranch') ?: 'develop'}",
+        "data_commit": "${ext.getParameterValue('dataCommit') ?: 'HEAD'}",
+        "run_beam": "false",
+        "run_jupyter" : "true",
+        "jupyter_token" : "${ext.getParameterValue('jupyterToken') ?: UUID.randomUUID().toString()}",
+        "shutdown_wait": "${ext.getParameterValue('jupyterShutdownWait') ?: 60 * 8}",
+        "storage_size": "${ext.getParameterValue('storageSize')}",
+        "shutdown_behaviour": "${ext.getParameterValue('shutdownBehaviour')}",
+        "storage_publish": ${"false".equalsIgnoreCase(ext.getParameterValue('s3Backup')) ? "false" : "true"}
+        }"""
+        logger.warn(pload)
+        HttpResponse result = makeJsonPostRequest("https://us-central1-beam-core.cloudfunctions.net/deploy_beam", pload)
+        logger.warn("response status: ${result.statusCode}, response message: ${result.statusMessage}, payload: ${result.content}")
+    }
+}
```

Comment:
Is that possible to just reuse `deployToGCE` task with correctly defined params (run_beam, run_jupyter)?